src/thunderbots/software/radio_communication/mrf_backend.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <limits>
+
+#include "ai/world/ball.h"
+#include "ai/world/team.h"
+#include "mrf/dongle.h"
+
+class MRFBackend
+{
+   public:
+    /**
+     * Creates a new MRFBackend.
+     * Automatically connects to the dongle upon initialization.
+     */
+    explicit MRFBackend();
+
+    ~MRFBackend();
+
+    /**
+     * IMPORTANT: Must be called in the main loop
+     * Allows libusb events on the dongle to complete.
+     */
+    void update_dongle_events();
+
+    /**
+     * Sends the given primitives to the backend to control the robots
+     *
+     * @param primitives the list of primitives to send
+     */
+    void sendPrimitives(const std::vector<std::unique_ptr<Primitive>>& primitives);
+
+    /**
+     * Updates the detected robots from vision.
+     *
+     * @param ft a vector of tuples of {robot id, robot location, robot orientation}
```

Comment:
could we use a slightly more verbose name? not really sure what `ft` stands for here.

Reply:
renamed